consult.el

```diff
@@ -1935,10 +1935,14 @@ See `multi-occur' for the meaning of the arguments BUFS, REGEXP and NLINES."
   (consult--forbid-minibuffer)
   (let* ((line (line-number-at-pos (point-min) consult-line-numbers-widen))
          (heading-regexp (concat "^\\(?:"
-                                 (if (boundp 'outline-regexp)
```

Comment:
If `outline-regexp` is bound and nil (which shouldn't happen), there will be an error in the ensuing `re-search-forward`. So I changed to `(or` (bound-and-true-p ...`